examples/morpheusvm/actions/transfer_test.go

```diff
@@ -124,3 +125,41 @@ func TestTransferAction(t *testing.T) {
 		tt.Run(context.Background(), t)
 	}
 }
+
+func BenchmarkSimpleTransfer(b *testing.B) {
+	require := require.New(b)
+	to := codec.CreateAddress(0, ids.GenerateTestID())
+	from := codec.CreateAddress(0, ids.GenerateTestID())
+	toBalanceKey := storage.BalanceKey(to)
+	fromBalanceKey := storage.BalanceKey(from)
+
+	transferActionTest := &chaintest.ActionBenchmark{
+		Name:  "SimpleTransferBenchmark",
+		Actor: from,
+		Action: &Transfer{
+			To:    to,
+			Value: 1,
+		},
+		CreateState: func() state.Mutable {
+			keys := make(state.Keys)
```

Comment:
It looks like this local variable is not used by anything that actually gets returned. This seems to be copy-pasted from the test above. Could you remove it from here and the test above? Running locally confirms it's not needed for the tests to pass.